### PR TITLE
Abracadabra

### PR DESCRIPTION
**Pull request by tannishmango**
Fixed TVL, cumulative values

### CODE REVIEW

File: subgraphs/abracadabra/schema.graphql [hunk 1]
```diff
@@ -278,6 +278,98 @@ type LendingProtocol implements Protocol @entity {
   marketIDList: [String!]!
 }
 
+type LendingProtocol implements Protocol @entity {
```
Comment (this-username-is-taken) [inline subgraphs/abracadabra/schema.graphql:281]: Should update schema to 1.2.1 (version)

**Comment by this-username-is-taken** [comment]
Next time you do updates like this, you should pull latest master and start a new branch, so it doesn't include all historical commits you had

File: subgraphs/abracadabra/schema.graphql (hunk 1 above)
Comment (this-username-is-taken) [inline subgraphs/abracadabra/schema.graphql:281]: This is copy/pasted twice?

File: subgraphs/abracadabra/src/common/getters.ts [hunk 2]
```diff
@@ -201,7 +206,7 @@ export function getOrCreateLendingProtocol(): LendingProtocol {
   LendingProtocolEntity = new LendingProtocol(getBentoBoxAddress(dataSource.network()));
   LendingProtocolEntity.name = "Abracadabra Money";
   LendingProtocolEntity.slug = "abracadabra";
-  LendingProtocolEntity.schemaVersion = "1.2.0";
+  LendingProtocolEntity.schemaVersion = "1.2.1";
   LendingProtocolEntity.subgraphVersion = "1.0.0";
```
Comment (this-username-is-taken) [inline subgraphs/abracadabra/src/common/getters.ts:210]: Should also update the subgraph version for every PR

File: subgraphs/abracadabra/src/common/metrics.ts [hunk 3]
```diff
@@ -299,7 +321,7 @@ export function updateMarketStats(
       ? priceUSD
       : getOrCreateToken(Address.fromString(getMIMAddress(dataSource.network()))).lastPriceUSD;
   market.save();
-  usageDailySnapshot.save();
+  usageHourlySnapshot.save();
```
Comment (this-username-is-taken) [inline subgraphs/abracadabra/src/common/metrics.ts:324]: nice catch!

File: subgraphs/abracadabra/schema.graphql (hunk 1 above)
Comment (tannishmango) [reply subgraphs/abracadabra/schema.graphql:281]: whoops lol

File: subgraphs/abracadabra/src/common/getters.ts (hunk 2 above)
Comment (tannishmango) [reply subgraphs/abracadabra/src/common/getters.ts:210]: gotcha

**Pull request merged**